x-pack/plugins/maps/public/routing/routes/list/maps_list_view.tsx

```diff
@@ -44,9 +52,30 @@ function navigateToNewMap() {
     path: MAP_PATH,
   });
 }
+interface MapsListViewState {
```

Comment:
Mind renaming MapsListViewState to just State

Reply:
Fixed this.